fiftyone/utils/annotations.py

```diff
@@ -407,13 +407,21 @@ def _build_label_schema(
         _return_type = _RETURN_TYPES_MAP[_label_type]
         _is_trackable = _is_frame_field and _return_type in _TRACKABLE_TYPES
 
-        if is_video and _return_type in _SPATIAL_TYPES and not _is_frame_field:
-            raise ValueError(
-                "Invalid label field '%s'. Spatial labels of type '%s' being "
-                "annotated on a video must be stored in a frame-level field, "
-                "i.e., one that starts with 'frames.'"
-                % (_label_field, _label_type)
-            )
+        if is_video and not _is_frame_field:
+            if not backend.supports_video_sample_fields:
```

Comment:
nit: I'd check for the second error first. If a user has provided a spatial label field, they most likely just forgot `frames.`, and the second error is more apropos for that